retwist/util/sentry.py

```diff
@@ -1,19 +1,29 @@
+from typing import Dict
+
 import sentry_sdk
 from twisted.python import log
 from twisted.python.failure import Failure
 from twisted.web.http import Request
 
 
-def add_request_context_to_scope(request, scope):
-    # type: (Request, sentry_sdk.Scope) -> None
+def add_request_context_to_scope(request, scope, context):
+    # type: (Request, sentry_sdk.Scope, Dict) -> None
     """
-    Add context information which is useful for debugging from a Twisted request to a Sentry scope.
+    Add error context information which is useful for debugging from a Twisted request to a Sentry scope.
     """
-    scope.set_extra("url", request.uri)
-    scope.set_extra("method", request.method)
-    scope.set_extra("headers", request.getAllHeaders())
-    scope.set_extra("query_string", request.uri.replace(request.path, b"").strip(b"?"))
-    scope.set_extra("data", request.args)
+    request_context = {
```

Comment:
@cristianc-ty looking at this, it would make more sense to me to add the default request context to the `JsonResource.get_context` parent method; the one which just returns an empty dict.

In fact, then you wouldn't have to pass on the request as a kwarg to `log.err` at all.

What do you think?

Reply:
In fact this was an early version of the implementation, but the reason I ended up with this version is to keep a backward compatible interface between JsonResource and the sentry log observer, that already had request in keyword param. Let me know how important is this aspect, if it is a no issue from my perspective we can refactor as you suggest.

Reply:
Ah, thanks for the context!

I don't think it's important to maintain backwards compatibility. I would even be OK with removing `add_request_context_to_scope` altogether if it doesn't really do much any more. Both changes were added quite recently, and are still available in 0.3 for anyone that depends on them.

I hope the vast retwist community will not be outraged :)

Reply:
Refactored.